pytest_django/fixtures.py

```diff
@@ -178,13 +178,19 @@ def transactional_db(request, django_db_setup, django_db_blocker):
 
 
 @pytest.fixture()
-def client():
-    """A Django test client instance."""
+def client_class():
```

Comment:
Can we name it `django_client_class`?
New fixtures should have the `django_` prefix (from/via pytest-django, not necessarily from Django).